edit-post/store/selectors.js

```diff
@@ -108,6 +108,23 @@ export function isFeatureActive( state, feature ) {
 	return !! state.preferences.features[ feature ];
 }
 
+/**
+ * Returns true if the the plugin item is pinned to the header.
+ * When the value is not set it defaults to true.
+ *
+ * @param  {Object}  state      Global application state.
+ * @param  {string}  pluginName Plugin item name.
+ * @return {boolean}            Whether the plugin item is pinned.
+ */
+export function isPluginItemPinned( state, pluginName ) {
+	const defaultValue = true;
+	const pinnedPluginItems = getPreference( state, 'pinnedPluginItems', {} );
+
+	return isUndefined( pinnedPluginItems[ pluginName ] ) ?
```

Comment:
Maybe leverage `get` and it's `defaultValue` argument?

```js
return get( getPreference( state, 'pinnedPluginItems', {} ), [ pluginName ], true );
```

https://lodash.com/docs/4.17.10#get